lucet-runtime/lucet-runtime-internals/src/region.rs

```diff
@@ -98,9 +101,18 @@ impl<'a> InstanceBuilder<'a> {
             region,
             module,
             embed_ctx: CtxMap::default(),
+            heap_memory_size: region.heap_memory_size_limit(),
         }
     }
 
+    /// Add a custom limit for the heap memory size to the built instance.
+    /// This call is not necessary if the default heap memory size is adequate
```

Comment:
```suggestion
    /// Add a custom limit for the heap memory size to the built instance.
    ///
    /// This call is not necessary if the default heap memory size is adequate
```

The blank line separates the short description from the rest in the rendered rustdoc